vignettes/s3-vector.Rmd

```diff
@@ -629,9 +629,7 @@ decimal2(10, c(0, 5, 99))
 
 vctrs provides two "proxy" generics that lets you control how your class determines equality and ordering:
 
-* `vec_proxy_equal()` specifies how to test the elements of your vector for equality. 
-  This proxy underpins `==` and `!=`, but also `unique()`, `anyDuplicated()`, 
-  and `is.na()`.
+* `vec_proxy()` is the main way for an arbitrary class that doesn't inherit from `vctrs_vctr` to be compatible with vctrs. It is used in most vctrs primitives, including equality comparison. This proxy thus underpins `==` and `!=`, `unique()`, `anyDuplicated()`, and `is.na()`.
```

Comment:
We need to mention this in the news too — it's technically a major change, although I doubt anyone has depended on it yet.

Reply:
I think the fact that you can use this to make an arbitrary class compatible with vctrs isn't relevant (or at least not primary here).

Reply:
Never say never. Easy change though, when the time comes. https://github.com/tidymodels/probably/blob/ff58ca94305bbc792fc126ae270beac103149a94/R/vctrs-compat.R#L304